test/components/flatcar-linux-update-operator/fluo_reboot_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build packet_fluo
+// +build disruptivee2e
+
+package fluo //nolint:testpackage
```

Comment:
Why not `fluo_test` ?

Reply:
There will be two packages in the same directory and I am not sure if it is a good thing or a bad thing.

Reply:
Also the package would be `github.com/kinvolk/lokomotive/test/components/flatcar-linux-update-operator/fluo_test` which is test and again test.

Reply:
I don't think it's a problem. Package name doesn't really matter here, but we should avoid adding `nolint` statements where possible.